Resources/DesktopModules/Admin/Dnn.PersonaBar/Modules/Dnn.Licensing/App_LocalResources/licensing.nl-NL.resx

```diff
@@ -154,13 +142,13 @@
     <value>DOCUMENTATIE CENTER</value>
   </data>
   <data name="GitHub">
-    <value>The open source DNN Platform project is managed on GitHub. Visit us there and get involved. Report issues, provide valuable feedback, and contribute.</value>
+    <value>&lt;div&gt;Het Open Source DNNlatform project is volledig beschikbaar op GitHub. Bezoek ons en raak betrokken. Meld problemen, geef terugkoppeling en draag bij aan de ontwikkeling van het platform&lt;/div&gt;</value>
   </data>
   <data name="GitHub.Header">
     <value>THE CODE</value>
   </data>
   <data name="Intro">
-    <value>DNN® ( formerly DotNetNuke® ) is the leading open source web content management platform (CMS) in the Microsoft ecosystem. The product is used to build professional looking and easy-to-use commercial websites, social intranets, community portals, or partner extranets. Containing dynamic content of all types, DNN sites are easy to deploy and update. The DNN Platform has been downloaded more than 8 million times and powers more than 750,000 websites globally. A community of more than 1 million members forms a powerful support network.</value>
+    <value>DNN® (voorheen DotNetNuke® ) is het leidende Open Source content management platform (CMS) in het Microsoft ecosysteem. DNN wordt gebruikt om professionele website te maken, social intranets, community portals, extranets en meer.&lt;/div&gt;&lt;div&gt;DNNPlatform is meer dan 8 milhoen keer gedownload en kent een activieve commuity van meer dan 1 miljoen leden.</value>
```

Comment:
<value>DNN® (voorheen DotNetNuke®) is het leidende Open Source content management platform (CMS) in het Microsoft ecosysteem. DNN wordt gebruikt om professionele websites te maken, social intranets, community portals, extranets en meer.&lt;/div&gt;&lt;div&gt;DNNPlatform is meer dan 8 miljoen keer gedownload en kent een actieve community van meer dan 1 miljoen leden.</value>